scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -610,14 +620,20 @@ export class DependencyLinker {
     }
   }
 
-  private linkHarmony(rootDir: string): LinkDetail | undefined {
+  private linkHarmony(rootDir: string, rootPolicy: WorkspacePolicy): LinkDetail | undefined {
     const name = 'harmony';
-    return this.linkNonAspectCorePackages(rootDir, name);
+    const packageName = `@teambit/${name}`;
+    const existInRootPolicy = rootPolicy ? !!rootPolicy.find(packageName) : false;
```

Comment:
I guess this would be simpler
```suggestion
    const existInRootPolicy = Boolean(rootPolicy?.find(packageName));
```

although it doesn't look like rootPolicy is nullable, so typescript might not accept it

Reply:
It used to be like this, but I had a problem so I changed it.
but later on I find that the problem was something else, and forgot to restore it to be like this)